packages/api-explorer/src/JsonForm/configureJsonEditor.js

```diff
@@ -12,7 +12,17 @@ function setDefaultCustomization (JSONEditor) {
     JSONEditor.defaults.editors[key] = class Customization extends JSONEditor.defaults.editors[key] {
       showEditJSON(){
```

Comment:
Does this method exist in all editors?

If not, we could try to see if is possible to make this changes only on the "object" editor

Reply:
I moved the changes in `objects-editor.js`!